tutorials/enricher/pom.xml

```diff
@@ -13,9 +13,9 @@
 
   <properties>
     <!-- change this if you desire to use a different version of Apex Core -->
-    <apex.version>3.4.0</apex.version>
+    <apex.version>3.6.0-dt20161215</apex.version>
```

Comment:
@amberarrow  Hmmm, will need to discuss this with @priteshm and others then. BTW, is it possible to get the dt versions by adding another maven repository?